src/.nuget/init/init.csproj.nuget.cache

```diff
@@ -0,0 +1,5 @@
+{
+  "version": 1,
```

Comment:
I do not think you meant to add all these files.

Reply:
oh goodness... i certainly did not. I'll fix up the last commit.